docs/Upgrading.md

```diff
@@ -51,6 +51,10 @@ If you have overwritten these methods in your callback controller to modify the
 update your app to use configurable option `config.custom_post_authenticate_tasks` instead. See [post authenticate tasks](/docs/shopify_app/authentication.md#post-authenticate-tasks)
 for more information.
 
+#### (v23.0.0) - Deprecated "ShopifyApp::JWT" class
```

Comment:
I'd also highlight that the API of the new one is a superset of the old one, so apps can safely search replace them if they ever reference the class directly, or not have to update at all if we return this token at any point.